limitador-server/sandbox/docker-compose-limitador-memory.yaml

```diff
@@ -27,4 +26,6 @@ services:
       - "18080:8080"
       - "18081:8081"
     volumes:
-      - ./limits.yaml:/opt/kuadrant/limits/limits.yaml
+      # the entire path needs to be mounted
+      # or inotify events are not triggered for updates on the mounted file
```

Comment:
This part?

Reply:
This is a comment to warn about some configuration that, for unknown reason to me yet, events are not triggered when the file changes. That happens exactly when the file is mounted. 

When the parent dir is mounted, then events are triggered whenever the limits file is updated.